modules/dbparser/radix.c

```diff
@@ -866,7 +866,7 @@ r_add_child_check(RNode *root, gchar *key, gpointer value, RNodeGetValueFunc val
 void
 r_add_pchild(RNode *parent, RNode *child)
 {
-  parent->pchildren = realloc(parent->pchildren, (sizeof(RNode *) * (parent->num_pchildren + 1)));
+  parent->pchildren = g_realloc(parent->pchildren, (sizeof(RNode *) * (parent->num_pchildren + 1)));
```

Comment:
Rather a note: g_malloc *can* return with `NULL` if len is 0 (well it is never a case here, but the commit message is misleading imho). Also `g_malloc` simple aborts the program if it cannot allocate.

Reply:
Rephrased, thanks.